cafebabel/migrations.py

```diff
@@ -0,0 +1,42 @@
+import json
+from datetime import datetime, timezone
+
+import click
+from cafebabel.users.models import Role
+from progressist import ProgressBar
+
+
+def iterate_over_json_file(filename):
+    with open(filename) as json_file:
+        return json.loads(json_file.read())
+
+
+def migrate_users(app, limit):
+    click.echo('Starting users migration.')
+    ds = app.user_datastore
+    old_users = iterate_over_json_file('users.json')
+    if limit:
+        old_users = old_users[:limit]
+    bar = ProgressBar(
+        total=len(old_users),
+        template=('{prefix} {animation} {percent} ({done}/{total}) '
+                  'ETA: {eta:%H:%M}'),
+        done_char='👤')
```

Comment:
Looks cool but Github reads it different ^^

Reply:
What do you mean? The user-kind-emoji is intended!